pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -190,7 +190,7 @@ for (int i = 0; i < 10; i++) {
         <description>
 The method Object.finalize() is called by the garbage collector on an object when garbage collection determines
 that there are no more references to the object. It should not be invoked by application logic.
-            
```

Comment:
Please try to avoid changing unrelated whitespaces. These changes are only distracting. If they are necessary, then they should be in a separate PR only dealing with whitepaces.